src/__init__.py

```diff
@@ -208,19 +208,24 @@ def create_item(self, name: str) -> Item:
         name = before_create_item(name, self, self.multiworld, self.player)
 
         item = self.item_name_to_item[name]
-        classification = ItemClassification.filler
+        filler = ItemClassification.filler
+        trap = ItemClassification.filler
+        useful = ItemClassification.filler
+        progression = ItemClassification.filler
+        progression_skip_balancing = ItemClassification.filler
```

Comment:
If you're applying filler to start, makes more sense to just apply flags as you go instead of accumulating them and applying them all at the end.

e.g.
```
classification = ItemClassification.filler

if "trap" in item and item["trap"]:
   classification |= ItemClassification.trap # or classification = classification | ItemClassification.trap
```
etc.

Reply:
Forgot to leave a reply here - I changed over to your first suggestion! Thank you for the input!